subsys/bluetooth/mesh/shell/shell_utils.c

```diff
@@ -37,24 +37,10 @@ static void model_instances_get(uint16_t id, struct shell_model_instance *arr, u
 	}
 }
 
-static uint8_t str2u8(const char *str)
-{
-	if (isdigit((unsigned char)str[0])) {
-		return strtoul(str, NULL, 0);
-	}
-
-	return (!strcmp(str, "on") || !strcmp(str, "enable"));
-}
-
-bool shell_model_str2bool(const char *str)
-{
-	return str2u8(str);
-}
-
-
-int shell_model_str2sensorval(const char *str, struct sensor_value *out)
+struct sensor_value shell_model_str2sensorval(const char *str, int *err)
```

Comment:
Also, I think maybe this entire thing can be replaced with a wrapper around str2dbl now?

Reply:
I'll take a look 👍